cmd/rook/ceph/cleanup.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ceph
+
+import (
+	"github.com/pkg/errors"
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dataDirHostPath      string
+	cleanDataDirHostPath bool
+)
+
+var cleanUpCmd = &cobra.Command{
+	Use:   "clean",
+	Short: "Starts the cleanup process on the disks after ceph cluster is deleted",
+}
+
+func init() {
+	cleanUpCmd.Flags().StringVar(&dataDirHostPath, "data-dir-host-path", "", "dataDirHostPath on the node")
+	cleanUpCmd.Flags().BoolVar(&cleanDataDirHostPath, "clean-data-dir-host-path", false, "whether dataDirHostPath should be cleaned")
+	flags.SetFlagsFromEnv(cleanUpCmd.Flags(), rook.RookEnvVarPrefix)
+	cleanUpCmd.RunE = startCleanUp
+}
+
+func startCleanUp(cmd *cobra.Command, args []string) error {
+
+	rook.SetLogLevel()
+	rook.LogStartupInfo(cleanUpCmd.Flags())
+
+	logger.Infof("starting cluster clean up")
+
+	context := rook.NewContext()
+
+	if cleanDataDirHostPath {
```

Comment:
What about checking `if dataDirHostPath != ""`? Then you don't need a separate variable to trigger the removal.